first_analysis.py

```diff
@@ -27,17 +27,30 @@
 plt.ylabel('Survived')
 plt.show()
 
-# running a logistic regression (survived ~ gender + fare)
-mod = smf.logit('survived_dummy ~ gender + fare', data = df).fit()
-
-display(mod.summary())
-
 
 # running a logistic regression (survived ~ gender + fare + age)
 mod = smf.logit('survived_dummy ~ gender + fare + age', data = df).fit()
 
 display(mod.summary())
 
 
+# creating a 3D plot (survived ~ gender + fare)
+survived = df[df['survived_dummy'] == 1]
```

Comment:
Isn't dummy already 0 and 1?   How about

```suggestion
survived = df[df['survived_dummy']]
```

Reply:
That will mean it includes both 0s and 1s thought? It should just be survived?

Reply:
Hmm - not quite - but it might select the element at position 0 and the element at position 1, many times, and that would be wrong, I agree.